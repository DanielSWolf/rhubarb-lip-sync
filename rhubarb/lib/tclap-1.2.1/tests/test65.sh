#!/bin/sh

# this tests whether all required args are listed as
# missing when no arguments are specified
# failure  
../examples/test12 -v "1 2 3" -v "4 5 6" -v "7 8 9" -v "-1 0.2 0.4" \
    > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test65.out; then
	exit 0
else
	exit 1
fi

