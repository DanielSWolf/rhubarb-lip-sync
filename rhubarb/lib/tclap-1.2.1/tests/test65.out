1 2 3 
4 5 6 
7 8 9 
-1 0.2 0.4 
REVERSED
-1 0.2 0.4 
7 8 9 
4 5 6 
1 2 3 
